components/x-privacy-manager/storybook/story-container.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import { Helmet } from 'react-helmet'
+
+import BuildService from '../../../.storybook/build-service'
+import { default as pkg } from '../package.json'
+import { PrivacyManager } from '../src/privacy-manager'
+
+const dependencies = {
+	'o-loading': '^4.0.0',
+	'o-message': '^4.0.0',
+	'o-typography': '^6.0.0'
+}
+
+/**
+ * @param {import("../typings/x-privacy-manager").PrivacyManagerProps} args
+ */
+export function StoryContainer(args) {
+	return (
+		<div className="story-container">
+			{dependencies && <BuildService dependencies={dependencies} />}
+			{pkg.style && (
+				<Helmet>
+					<link rel="stylesheet" href={`components/x-privacy-manager/${pkg.style}`} />
```

Comment:
since we know that the `package.json` for `x-privacy-manager` does have a `style` property, could we remove the `pkg.style` check and reference the css directly?

```
<link rel="stylesheet" href={`components/x-privacy-manager/dist/privacy-manager.css`} />
```

This might have gotten carried over from the pre-migration code since `pkg` was being used when the stories where being created dynamically with the `buildStory` function

Reply:
Seems highly likely: I'll use your updated code 👍

Reply:
Done & done